tests/integration/data/hana_to_file_default.yml

```diff
@@ -3,23 +3,19 @@
 # The original source code can be found at https://github.com/acryldata/datahub/blob/master/metadata-ingestion/tests/integration/hana/hana_to_file.yml
 # And the original Apache 2 license is available at https://github.com/acryldata/datahub/blob/master/LICENSE
 
-run_id: hana-test
+#run_id: hana-test
```

Comment:
I don't think you should edit this recipe, it is specifically being used for you integration tests. If you want a recipe to use for manual testing, just create a new one in the project root, don't abuse and break this integration test which is supposed to write the output to a file. Please, make sure to run the full test suite locally before pushing a new PR